templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -7,7 +7,13 @@ metadata:
 {{ include "harbor.labels" . | indent 4 }}
     component: chartmuseum
 spec:
+  {{- if and .Values.persistence.enabled (eq .Values.persistence.imageChartStorage.type "filesystem") }}
+  replicas: 1
```

Comment:
Why the replicas must be 1 here?

Reply:
If `imageChartStorage.type` is `filesystem` then storage is done as regular files, and you can't share one ("dumb") filesystem across multiple replicas, so we have to make sure that no more than one replica is created.

Reply:
The `imageChartStorage.type` is an application level configuration item, it can be `filesystem` or other cloud storage. But even though it's `filesystem`, it's backend PV can be shared between multiple nodes. So the `replicas: 1` isn't needed

Reply:
Unfortunately this would require PVC with ReadWriteMany which is working only on small subset of volume plugins.

Reply:
Yes. So setting replicas to 1 or not is decided by users